### DISCUSSION TITLE
Interface for sending data to LYWSD03MMC

### DISCUSSION THREAD

**Discussion by yuriytkachenko-ideals** [custom-components/ble_monitor#279] [Ideas]
Is it possible to provide an additional interface for sending data to LYWSD03MMC device. For example Python script, or any other command which can be executed from Home Assistant, Node-Red, or any other system. Thanks!

**Comment by Ernst79** [comment]
What kind of data do you want to send to LYWSD03MMC?

**Reply by yuriytkachenko-ideals** [reply]
For example, I have a Co2 sensor, without a display. Good to have the ability to send sensor data to LYWSD03MMC display and show it as an additional parameter

**Reply by yuriytkachenko-ideals** [reply]
I saw that it possible to do it from HTML form. Good to have any other interface for it.

**Reply by Ernst79** [reply]
ah, for sensors with the ATC firmware (https://github.com/pvvx/ATC_MiThermometer#interface-for-receiving-and-displaying-data-on-the-lcd).

I'm afraid this won't be done soon/easy (for me :-). Our component is designed to listen only, it can't send anything (at the moment). So it will need quite some work. I'm afraid I won't do this myself, due to the lack of time, but any contributions are welcome.

**Reply by yuriytkachenko-ideals** [reply]
It`s a pity, thank you!
